eventuate-common-common-spring-jdbc/src/main/java/io/eventuate/common/common/spring/jdbc/EventuateSpringJdbcStatementExecutor.java

```diff
@@ -17,6 +21,26 @@ public EventuateSpringJdbcStatementExecutor(JdbcTemplate jdbcTemplate) {
     this.jdbcTemplate = jdbcTemplate;
   }
 
+  @Override
+  public Long insertAndReturnGeneratedId(String sql, Object... params) {
```

Comment:
This should return `long` since it cannot return null

Reply:
ok, I will fix